### DISCUSSION TITLE
Is it Possible to Schedule Remote Backups?

### DISCUSSION THREAD

**Discussion by twgonzalez** [supabase/supabase#10292] [Questions]
Are there any best practices for scheduling remote backups? Is it possible to run a cron job that executes pg_dump and sends the file to a remote server? Or are there any other platforms/services that would allow me to remotely execute and store backups via the instance URI and credentials? 

My concern is that the 7 day window provided on a pro account is not enough for our production servers, and we also need a more fail safe solution for storing backups remotely on an automated basis.

**Comment by monicakh** [comment]
Yes, you can schedule your own backups from any location at any time.

1. You must first configure a server (or a local machine) to accomplish this. It could be a remote VM or a laptop or desktop computer at home.
2. Create a script or batch file to perform the dump (you can look at our migration guide to get the dump script.)
3. You must configure a cron job or other scheduler (Windows has its own scheduler) to run your backup job(s) according to your specifications.
